colors/focus.lua

```diff
@@ -0,0 +1,34 @@
+-- color theme from the focus text editor
+
+local style = require "core.style"
+local common = require "core.common"
+
+style.caret_width = 2
```

Comment:
Is this needed?
Might be a good idea to either remove it or do:
```suggestion
style.caret_width = common.round(2 * SCALE)
```
but this is the default anyways, so might as well remove it.